zerver/openapi/zulip.yaml

```diff
@@ -14442,6 +14453,16 @@ paths:
                           client capability and access the `user_settings` object instead.
 
                           [capabilities]: /api/register-queue#parameter-client_capabilities
+                      typing_notifications_enabled:
+                        type: boolean
+                        description: |
+                          Present if the user has chosen to receive typing notifications from
+                          other users.
+
+                          By default user receives typing notifications from other users.
+
+                          **Changes**: New in Zulip 9.0 (feature level 251). Previously, there
+                          was no such option for users to disable typing notifications.
```

Comment:
I think we should be clearer here, since there are options to disable sending them.

Reply:
How about `**Changes**: New in Zulip 9.0 (feature level 251). Previously, there was no such option wherein users could decide whether to receive the  typing notification events of other users or not.`

Reply:
Simpler would be "Previously, there were only options to disable sending typing notifications."

Reply:
updated